src/app/resources/resources.component.html

```diff
@@ -1,25 +1,66 @@
-<div>
-  <button mat-raised-button routerLink="add" color="primary" i18n>Add Resource</button>
-</div>
-<div>
-  <mat-form-field>
-    <input matInput (keyup)="applyResFilter($event.target.value)" i18n-placeholder placeholder="Search">
-  </mat-form-field>
-</div>
+<mat-toolbar>
+  <mat-toolbar-row>
+    <button mat-icon-button (click)="back()"><mat-icon>arrow_back</mat-icon></button>
+    <span i18n>Library</span>
+    <span class="toolbar-fill"></span>
+    <mat-icon>search</mat-icon>
+    <mat-form-field class="font-size-1">
+      <input matInput (keyup)="applyResFilter($event.target.value)" i18n-placeholder placeholder="Type text to search...">
+    </mat-form-field>
+  </mat-toolbar-row>
+
+  <mat-toolbar-row class="primary-color font-size-1">
+    <mat-checkbox i18n>Click to select all</mat-checkbox>
+    <span class="toolbar-fill"></span>
+    <mat-icon>folder</mat-icon><span i18n>View Details</span>&nbsp;&nbsp;
+    <mat-icon>library_add</mat-icon><span i18n>Add to Library</span>&nbsp;&nbsp;
+    <mat-icon>feedback</mat-icon><span i18n>Feedback</span>&nbsp;&nbsp;
+    <mat-icon>delete_forever</mat-icon><span i18n>Delete</span> &nbsp;&nbsp;
+    <button mat-mini-fab routerLink="add" ><mat-icon>add</mat-icon></button>
+  </mat-toolbar-row>
+</mat-toolbar>
+
 <mat-table #table [dataSource]="resources">
+  <ng-container matColumnDef="select">
+    <mat-header-cell *matHeaderCellDef>
```

Comment:
There is no header so this can be removed (the `mat-header-row` element is required for all tables, but not `mat-header-cell`)